pages/workflow/overview.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: 'guide'
+title: 'Avo Workflow'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+
+The Avo workflow is the recommended approach for a team working on an analytics release. The workflow is helpful both when new analytics tracking is being implemented, as well as when existing tracking is being updated.
+
+The Avo workflow for analytics releases is inspired by the general best practices of software development using branched change management, peer-reviews, and approval workflows.
+
+1. **Plan** analytics update in Avo branch
+2. **Review** suggestions in Avo
+3. **Share** implementation instructions with developers via Avo
+4. **Implement** analytics events
+5. **Validate** implementation using Avo
+6. **Merge** Avo branch and **Publish** schema to downstream schema registries
+
+<center>
+  <img
+        width="100%"
+        src={require('../../images/avo-workflow.png')}
+        alt="The Avo Workflow: Plan, review, implement, validate and publish your tracking changes"
+  />
+</center>
```

Comment:
HMW use the same Zoom image thing as we have on the docs front page today 🤔

Reply:
zoom is now on every image.